assets/src/components/block-navigation.js

```diff
@@ -85,8 +90,10 @@ export default compose(
 		const { getCurrentPage, isReordering } = select( 'amp/story' );
 		const { getBlockOrder, getBlocksByClientId, getSelectedBlockClientId } = select( 'core/editor' );
 
+		const blocks = getCurrentPage() ? getBlocksByClientId( getBlockOrder( getCurrentPage() ) ) : [];
+
 		return {
-			elements: getCurrentPage() ? getBlocksByClientId( getBlockOrder( getCurrentPage() ) ) : [],
+			blocks: blocks.filter( ( { name } ) => ALLOWED_MOVABLE_BLOCKS.includes( name ) ),
```

Comment:
Curious why not display it in the Block Navigator, it's quite convenient for selecting blocks. At this moment it's not possible to use the navigator for moving blocks around anyway.

Or is it because the logic doesn't automatically force the CTA block as last? In this case that makes sense at this moment.

Reply:
> Or is it because the logic doesn't automatically force the CTA block as last? In this case that makes sense at this moment.

Yup, that.